src/views/Home.jsx

```diff
@@ -37,6 +37,13 @@ export function Home({ makeNewList, joinList, handleError, joinListErrorMsg }) {
 						type="text"
 						value={inputValue}
 						onChange={(event) => setInputValue(event.target.value)}
+						pattern="[a-zA-Z]+ [a-zA-Z]+ [a-zA-Z]+"
+						onInvalid={(e) =>
+							e.target.setCustomValidity(
```

Comment:
This is the first I learn about the setCustomValidity method from the HTMLElement. This is a great way to add validation on the client side to avoid incorrect, and possibly expensive API calls with invalid data.

Reply:
I am curious about the choice to put the event handler inline here as an anonymous function. Are there benefits to doing it here rather than pointing towards a named function?